app/components/MineralCard/MineralCard.tsx

```
@@ -85,13 +85,13 @@ export default function MineralCard({ index, mineral, isVisible } : { index: num
                   {mineral.name}
                 </InternalLink>
               </div>
-              <div className="flex flex-wrap items-center gap-1">
+              <div className="flex flex-wrap items-center">
                 {!!formulas.length && (
                   <h2 className="break-words max-w-full text-font-primary" dangerouslySetInnerHTML={{ __html: getRelevantFormula(formulas).formula }}></h2>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)